com.unity.ml-agents/Editor/MLAgentsSettingsProvider.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+using System.Linq;
+using System.IO;
+using UnityEngine;
+using UnityEditor;
+#if UNITY_2019_4_OR_NEWER
+using UnityEngine.UIElements;
+#else
+using UnityEngine.Experimental.UIElements;
```

Comment:
we can get dinged for this in package validation.  They don't want the use of any experimental APIs in package code.

Reply:
this may be ok once we bump our min spec to 2019.4 in the next few weeks

Reply:
I'll target this PR to v2 and remove all of the flags then